### DISCUSSION TITLE
Dificuldades enfrentadas ao tentar usar a ferramenta

### DISCUSSION THREAD

**Discussion by Marcos-P-S-S-A** [jrief/django-formset#81] [General]
Boa tarde, primeiramente, parabéns pelo projeto! Me considero inciante no django, porém, consegui utilizar todas as funcionalidades da biblioteca pertinentes às minhas necessidades, exceto quando o tema envolve processamento e validação de formulários. TO-DOS os problemas abordados na introdução da documentação se verificam e eu achei as soluções propostas incríveis! Contudo, algumas barreiras estão me impedindo de progredir, tentei usar os exemplos da documentação e todas as tentativas resultaram em erros. Por isso, deixo aqui algumas perguntas e ao final um apelo. 

Pergunta 1: O django-formset 1.0 é compatível com a versão 3.11 do python e 4.2 do django? Eu sei que na documentação está aparente que não, mas mesmo assim, perguntei pra reforçar.

Pergunta 2: Na parte da instalação, li: "Se a estrutura CSS a ser usada for instalada via npm, sugiro configurar o caminho de pesquisa como"
`{% load static %}
`STATICFILES_DIRS = [
    ('node_modules', BASE_DIR / 'node_modules'),
]`.
Desculpe a minha ignorância, mas porque 'node_modules'? A minha configuração está assim:
`STATICFILES_DIRS = [
    Path.joinpath(BASE_DIR, 'static')
]`.
Inclusive, o meu projeto chama outros arquivos de dentro do 'static', o bootsrap5 eu chamo assim: <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"...

Pergunta 3-4: A depender da resposta da pergunta 2, quais arquivos devo ter na minha pasta 'static'? Como baixá-los?

Apelo: Poderia criar um tutorial dinâmico explicando categoricamente como trabalhar com ModelForms usando django-formset, apontando todas as importações e exemplificando com modelos de arquivos: models, forms, views, urls e templates.html?

Sem mais para o momento, coloco-me à disposição para prestar esclarecimentos adicionais e/ou sanar quaisquer dúvidas advindas. Desde já agradeço!

**Comment by jrief** [comment]
Please use English!
